Scalar.Common/Platforms/POSIX/POSIXPlatform.cs

```diff
@@ -327,5 +350,37 @@ private static bool TryGetEnvironmentVariable(string name, out string val, out s
             error = null;
             return true;
         }
+
+        private static class NativeMethods
+        {
+            // Definitions from
+            // /Library/Developer/CommandLineTools/SDKs/MacOSX.sdk
```

Comment:
There are no more definitions here from that header.

```suggestion
```

Reply:
Fixed; I merged this comment into the one in `MacPlatform.cs` where the `PATH_MAX` value is defined.